addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -238,7 +238,7 @@ def _compute_amount(self):
     def _compute_from_account_move_ids(self):
         for sheet in self:
             if sheet.payment_mode == 'company_account':
-                if sheet.account_move_ids:
+                if sheet.account_move_ids.filtered(lambda move: move.state != 'draft'):
```

Comment:
what about canceled moves? I think you need to filter them out as well, or the computation will be wrong
```suggestion
                if sheet.account_move_ids.filtered(lambda move: move.state == 'posted'):
```